lib/openglvis.hpp

```diff
@@ -105,6 +109,39 @@ class VisualizationScene
    // How to scale the visualized object(s)
    double xscale, yscale, zscale;
 
+   SdlWindow * wnd;
+
+   glm::mat4 _projmat;
+
+   enum {
+      BG_BLK = 0,
+      BG_WHITE = 1
+   } _background;
+
+   const Material BLK_MAT =
+   {
+      { 0.0, 0.0, 0.0, 1.0 },
```

Comment:
This doesn't compile for me (Ubuntu 16.04, gcc 5.4.0) without double braces. Can't push the fix though.